types/appointment.d.ts

```diff
@@ -0,0 +1,74 @@
+declare module 'skedify-sdk' {
+  export type AppointmentState =
+    | 'incoming'
+    | 'outgoing'
+    | 'completed'
+    | 'cancelled'
+    | 'accepted'
+    | 'occurred'
+
+  export interface Appointment extends Omit<BaseFields, 'enterprise_id'> {
```

Comment:
Q: Why is uuid not added to the Appointment?

Q: Why is listing_uuid not added to the Appointment?

Reply:
Added those